tModPorter/tModPorter.Tests/TestData/SimpleRenamedVanillaMembersTest.Expected.cs

```diff
@@ -0,0 +1,44 @@
+using Terraria;
+using Terraria.Chat; // ChatHelper
+using Terraria.ID;
+using Microsoft.Xna.Framework;
+
+public class SimpleRenamedVanillaMembersTest
+{
+	void Method() {
+		var dresserX = Main.interactedDresserTopLeftX;
```

Comment:
You could try adding some of these to `Config.cs` yourself! (if you're interested). It doesn't yet have the support for renames across types, but for all the simple stuff that's just renamed within the same type, it should be one line to add!